test/unit/credit_card_test.rb

```diff
@@ -193,6 +193,7 @@ def test_should_display_number
   def test_should_correctly_identify_card_brand
     assert_equal 'visa',             CreditCard.brand?('4242424242424242')
     assert_equal 'american_express', CreditCard.brand?('341111111111111')
+    (222100..272099).each {|bin| assert_equal 'master', CreditCard.brand?(bin.to_s + '0089600203'), "Failed with BIN #{bin}"}
```

Comment:
If we don't need it to pass Luhn here (do we?), then why not just fill with '1111111111'?